dotcom-rendering/src/components/ScrollableSmall.importable.tsx

```diff
@@ -35,14 +35,17 @@ export const ScrollableSmall = ({
 	aspectRatio,
 	sectionId,
 }: Props) => {
+	const mobileBottomCards = [1, 3];
```

Comment:
Maybe it would be useful to have a comment to explain the layout, somewhere in this file. It might help with the mental load when reading this file